docs/modules/publisher-commandline.md

```diff
@@ -379,6 +379,14 @@ The `Alternative` field, where present, refers to the CLI argument applicable in
                                       Mode: Standalone and Orchestrated
                                       Type: boolean
                                       Default: true
+            RejectUnknownRevocationStatus=VALUE
+                                      OPC UA Client Security Config - aReject chain validation 
+                                      with CA certs with unknown revocation status, e.g.when the
```

Comment:
There should be a space between `e.g.` and `when` in `e.g.when`.

Reply:
done